src/server/pps/pretty/pretty.go

```diff
@@ -186,6 +189,78 @@ Job Counts:
 	return nil
 }
 
+// PrintDatumInfoHeader prints a file info header.
+func PrintDatumInfoHeader(w io.Writer) {
+	fmt.Fprint(w, "ID\tSTATUS\tTIME\t\n")
+}
```

Comment:
I really think you should put more info here given that the info is interesting and it doesn't seem like we've got enough columns here to fill the horizontal width of most people's terminals. I feel like we might as well use the space if it's there.